CHANGELOG-API.md

```diff
@@ -7,6 +7,11 @@ If you are looking for user-facing changes, check out [CHANGELOG.md](./CHANGELOG
 
 <!-- next version -->
 
+## v0.97.0
```

Comment:
please refer to this https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-a-changelog-entry to create changelog entry